Userland/Libraries/LibX86/Instruction.h

```diff
@@ -202,6 +203,7 @@ enum InstructionFormat {
 };
 
 static constexpr unsigned CurrentAddressSize = 0xB33FBABE;
+static constexpr unsigned CurrentOperandSize = 0xB33FB00F;
```

Comment:
This seems duplicate

Reply:
`CurrentAddressSize` is the magic number to use the address size as immediate size (used by moffset and near branches). `CurrentOperandSize` is for using the operand size (used by movsxd and affected by rex.w)

Reply:
Oh I can't read...